src/components/UserSatisfactionPoll.vue

```diff
@@ -21,25 +21,25 @@
       | We're really happy to hear you are enjoying ActivityWatch, but we think we can do even better! To help us help you, here are a few things you can do:
       ul
         li
-          | Support us on #[a(href="https://www.patreon.com/erikbjare") Patreon] or #[a(href="https://opencollective.com/activitywatch") Open Collective]
-        li 
-          | If you're using ActivityWatch in the workplace, consider asking your employer to support us!
-        li 
-          | Tell your friends and coworkers! Post about it on social media, we are on #[a(href="https://twitter.com/ActivityWatchIt") Twitter] and #[a(href="https://www.facebook.com/ActivityWatch") Facebook]
-        li 
+          | Support us on #[a(href="https://www.patreon.com/erikbjare") Patreon] or #[a(href="https://opencollective.com/activitywatch") Open Collective] (or by #[a(href="https://activitywatch.net/donate/") other donation methods])
+        li
+          | Using ActivityWatch at work? Consider asking your employer to support us!
+        li
+          | Post about it on social media and tell your friends and colleagues! We are on #[a(href="https://twitter.com/ActivityWatchIt") Twitter] and #[a(href="https://www.facebook.com/ActivityWatch") Facebook]
+        //li
           | Sign up for the newsletter
-        li 
+        li
           | Vote for new features on the #[a(href="https://forum.activitywatch.net/c/features") forum]
-        li 
+        //li
           | Fill out the #[a(href="https://forms.gle/q2N9K5RoERBV8kqPA") feedback form]
-        li 
-          | Rate us on #[a(href="https://alternativeto.net/software/activitywatch/") AlternativeTo]
-        li 
-          | Star us on #[a(href="https://github.com/ActivityWatch/activitywatch") GitHub]
+        li
+          | Rate us on #[a(href="https://alternativeto.net/software/activitywatch/") AlternativeTo] and #[a(href="https://play.google.com/store/apps/details?id=net.activitywatch.android") Google Play Store].
+        li
+          | Join our #[a(https://discord.gg/vDskV9q) Discord server].
```

Comment:
Huh, we have a discord server?

Reply:
Yeah, I'm testing it out. Tweeted about it a couple of days ago.